Foundation/Host.swift

```diff
@@ -88,9 +95,13 @@ open class Host: NSObject {
             }
             ifa = ifaValue.ifa_next
         }
+#endif
     }
     
     internal func _resolve() {
+#if os(Android)
+    return
```

Comment:
It looks like the indentation of this return doesn't line up - it should be one indent to the right, I think. @xwu can you confirm?

Reply:
Yeah, `return` needs four additional preceding spaces.